build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -49,10 +49,19 @@ def execute_cmd(args: Sequence[str],
   Raises:
     CalledProcessError if the command fails.
   """
+  cmd = " ".join(args)
   if verbose:
-    cmd = " ".join(args)
     print(f"cmd: {cmd}")
-  return subprocess.run(args, check=True, text=True, **kwargs)
+  try:
+    return subprocess.run(args, check=True, text=True, **kwargs)
+  except subprocess.CalledProcessError as exc:
+    print((f"\n\nThe following command failed:\n\n{cmd}"
+           f"\n\nReturn code: {exc.returncode}\n\n"))
+    if exc.stdout:
+      print(f"Stdout:\n\n{exc.stdout}\n\n")
```

Comment:
Done.  (Coming from a hg background, I find https://github.com/andrewshadura/git-crecord useful)

Reply:
(one of the commit appears duplicated in the commits list, really the 2nd one is just a 1-line fixup that I forgot to squash. not worth re-force-pushing for that).

Reply:
This PR is small enough that it's okay to edit history after sending for review, but also note that GitHub's review UI does not have a "view changes since I last commented" view. When you force push over changes editing the history, reviewers can have a hard time spotting what changed / what they already reviewed. It's usually most helpful in my experience to just push new commits (possibly using `--fixup`: https://fle.github.io/git-tip-keep-your-branch-clean-with-fixup-and-autosquash.html), then [auto]squash after review and prior to merging.

Reply:
Wait, I do try to avoid force-pushing as much as possible, but here we discussed slicing a commit into smaller commits. How could I do that without force-pushing?  Should I have first committed the reverse-commit of the original, then added on top?

Reply:
"prior to sending your PR for review" ;) 
(again, this one is small enough that it's fine, just sharing some tips for some ways to make GitHub reviews in general go more smoothly)